.github/workflows/5_builderpackage_indexer.yml

```
@@ -110,21 +116,32 @@ on:
 #   | https://docs.github.com/en/actions/learn-github-actions/expressions#example
 
 jobs:
-  matrix:
-    name: Set up matrix
+  setup:
+    name: Set up variables
     runs-on: ubuntu-24.04
     outputs:
-      matrix: ${{ steps.setup.outputs.matrix }}
+      matrix: ${{ steps.matrix.outputs.matrix }}
       previous_version: ${{ steps.previous_release.outputs.previous_version }}
     steps:
-      - id: setup
+      - id: matrix
         run: |
           matrix=$(jq -cn \
               --argjson distribution '${{ inputs.distribution }}' \
               --argjson architecture '${{ inputs.architecture }}' \
               '{distribution: $distribution, architecture: $architecture}'
           )
           echo "matrix=$matrix" >> $GITHUB_OUTPUT
+      - uses: actions/checkout@v4
+      - id: previous_release
+        name: Get latest Wazuh release from GitHub
+        run: |
+          CURRENT="$(bash packaging_scripts/product_version.sh)"
+          PREVIOUS=$(curl -s https://api.github.com/repos/wazuh/wazuh/releases | jq -r '.[].tag_name' | grep -vE 'alpha|beta|rc' | sed 's/^v//' | grep -E '^4\.[0-9]+\.[0-9]+$' | \
```

Replace with wazuh-indexer
```suggestion
          PREVIOUS=$(curl -s https://api.github.com/repos/wazuh/wazuh-indexer/releases | jq -r '.[].tag_name' | grep -vE 'alpha|beta|rc' | sed 's/^v//' | grep -E '^4\.[0-9]+\.[0-9]+$' | \
```